spec/models/run_spec.rb

```diff
@@ -294,7 +294,7 @@
     end
 
     it 'reports its total playtime' do
-      expect(run.total_playtime_ms(Run::REAL)).to eq 22_450_446
+      expect(run.total_playtime_ms).to eq 23_676_000
     end
```

Comment:
If I had to guess it would be that livesplit-core uses more information to get total playtime then just summing the histories like I had implemented before.  Cryze might be able to provide more information though.

Reply:
We iterate all the attempts and try this in the following order (taking the first one that matches):
1. If there's a started and ended, the diff is the playtime for that attempt
2. If there's a real time final time for the attempt we use that
3. If we can't extract either out of the attempt history, sum together the realtime segment times for the attempt in the segment histories.

And then add all of these together for all the attempts.

Reply:
Oh and we also subtract the pause times if that is stored in the attempt history for the attempt.